dev-docs/pb7-notes.md

```diff
@@ -0,0 +1,77 @@
+
+In the Prebid 5 release notes, it was noted that publishers should no longer use publisherDomain as a setConfig parameter, and instead prefer PageURL. Adapters no longer read from this location. Also, the object presented to bidders with the page url and that page's referring page url is essentially redone, and our goal is that adapters can rely on ortb2.site.page and ortb2.site.ref, with flags for when top is not reached, the canoncical link is used, setConfig('pageUrl') is used, or when window.location.href is used for ortb2.site.page. Many bidders had disparate logic for this and the refererInfo.referer seemed to have different meanings in different contexts.
+
+Also, Bid Adapters (not all adapters) no longer have access to storage manager unless explicitly whitelisted by the publisher. We believe bidding functionality should rarely if ever need this access and that this extra functionality included in bid adapters must be consented to by the installer in their configuration. The following bid adapters' unit tests were affected by this change: Adagio, Adnuntius, AP Stream, Concert, Conversant, Craft, Criteo, E-Planning, Invibes, Kargo, Quantcast, Trion, Unicorn, and Vidazoo. Adapters simply setting a random identifier in the first party local storage or cookie should consider if the popular shared id user id submodule can fully achieve this functionality for them. See storageAllowed in the [Publisher API Reference]({{site.baseurl}}/dev-docs/publisher-api-reference.html) 
+
+Adapters are also now not allowed to return alternate bidder codes unless whitelisted by the publisher. See allowAlternateBidderCodes in the [Publisher API Reference]({{site.baseurl}}/dev-docs/publisher-api-reference.html).
+
+Finally, adapters known to use http1 bidding endpoints may now have notices appended to their documentation. 
+
+## Regulatory and consent strings
+
+TCF1 is no longer supported by the Consent Management (GDPR) module. The Consent Management USP module now defaults to framework 'iab' and will work without configuration if installed and a functioning __uspapi() exists on the page. It will also set ortb2.regs.ext.gpc to 1 if GPC headers indicate the user has opted out, and the third character of the [US Privacy string](https://github.com/InteractiveAdvertisingBureau/USPrivacy/blob/master/CCPA/US%20Privacy%20String.md) will not be changed, but publishers may choose to have the string Prebid.js sees affected by GPC headers.
```

Comment:
```suggestion
TCF1 is no longer supported by the Consent Management (GDPR) module. The Consent Management USP module now defaults to framework 'iab' and will work without configuration if installed and a functioning `__uspapi()` exists on the page. It will also set ortb2.regs.ext.gpc to 1 if GPC headers indicate the user has opted out, and the third character of the [US Privacy string](https://github.com/InteractiveAdvertisingBureau/USPrivacy/blob/master/CCPA/US%20Privacy%20String.md) will not be changed, but publishers may choose to have the string Prebid.js sees affected by GPC headers.
```